src/main/java/network/brightspots/rcv/Logger.java

```diff
@@ -273,6 +273,17 @@ public void publish(LogRecord record) {
             }
           }
 
+          private boolean shouldIgnore(LogRecord record) {
+            // On Windows, scrollToBottom can trigger a log message in VirtualFlow.java that causes:
```

Comment:
Maybe mention that this is specific to JavaFX? `VirtualFlow.java` on its own is confusing.